woocommerce-myparcel.php

```diff
@@ -31,6 +31,10 @@
 
 require(plugin_dir_path(__FILE__) . 'vendor/autoload.php');
 
+ini_set('display_errors', '1');
+ini_set('display_startup_errors', '1');
+error_reporting(E_ALL);
```

Comment:
Ik denk niet dat dit wenselijk is want dit heeft global effects op alle error settings voor alle scripts van iemand die onze plugin installeert.

Reply:
Ai! Dat hoort zeker niet, ga ik weghalen!